### DISCUSSION TITLE
Custum Suricata Rules

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#2052] [Unsupported Versions] [answered]
Hello All,

You write custum rules in /opt/so/rules/nids/local.rules and you state.apply them. When you check than the local.rules file, it is empty.

Now my question is are these rules added or is the document completely overwritten in the suricata docker?

Where can i find the custum rules in the suricata docker for checking this?

Regards 
Bart

**Comment by weslambert** [comment]
They should be compiled into `all.rules`.

**Reply by sleepingbel** [reply]
Hello Wes,

I do not find the rules in the all.rules file, I imported the ICMP Anomaly Detection rules.
Where can i find them on the suricata docker? I have to delete some rules, too many false positive.
Or can I disable them like the ET Open Rules in the sls file.

ISO install, updated to 2.3.10

Regards 
Bart

**Reply by TOoSmOotH** [reply]
I updated the documentation so that it points to all.rules. You would still manage disabling the same way through the sls file.

**Reply by sleepingbel** [reply]
Thanks, I Placed them yesterday in the sls file and it works

To continue with the rules not being in the all.rules file, could this file be overwritten during a rule update?
If the all.rules file is overwritten, your added rules are gone.

Thx for the help
Regards 
Bart

**Comment by dougburks** [comment, marked as answer]
I've rewritten https://docs.securityonion.net/en/2.3/local-rules.html to hopefully make this more clear.
